### DISCUSSION TITLE
Adding html tag like link in hotspot

### DISCUSSION THREAD

**Discussion by micker** [google/model-viewer#4474] [Ideas]
Is it possible to render html in hotspot ? or can you add it ?

**Comment by elalish** [comment]
Absolutely, a hotspot literally *is* any HTML element that you assign to a `hotspot-*` slot. It can also contain any amount of nested child elements.

**Reply by micker** [reply]
Yep but anything IS render that display <a href... and not link for exemple
Need to add something ?

**Reply by elalish** [reply]
Can you rephrase? I don't quite follow. Maybe an example HTML you tried and images of what you expected vs what you got?

**Reply by micker** [reply]
I test in editor démo try to add HTML tag in hot spot. Notrhing thas render like HTML its just display plain text
